### PR TITLE
Get module 1 to start running 

### PR DESCRIPTION
**Pull request by rando2**
I made some additions that I think will allow this to run by including the ranges of parameters to optimize over

### CODE REVIEW

File: 1.initial-k-sweep/analysis.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ PARAM_FILE_PREFIX='config/z_parameter_sweep_'
 PMACS_FILE='../config/pmacs_config.tsv'
 PYTHON_PATH='python'
 ALGORITHMS=( 'tybalt' 'adage' )
-DATASETS=( 'TCGA' 'TARGET' 'GTEX' 'GREIN_human')
+DATASETS=( 'GREIN_human')
```
Comment (rando2) [inline 1.initial-k-sweep/analysis.sh:19]: We probably don't want to run it for the other three datasets anymore!

File: 1.initial-k-sweep/config/z_parameter_sweep_adage_GREIN_human.tsv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+parameter	sweep_values
```
Comment (rando2) [inline 1.initial-k-sweep/config/z_parameter_sweep_adage_GREIN_human.tsv:1]: Copied from GTEX (biggest dataset in the original project)

File: 1.initial-k-sweep/config/z_parameter_sweep_tybalt_GREIN_human.tsv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+parameter	sweep_values
```
Comment (rando2) [inline 1.initial-k-sweep/config/z_parameter_sweep_tybalt_GREIN_human.tsv:1]: Copied from GTEX
https://github.com/greenelab/BioBombe/pull/64#issuecomment-445255025

File: 1.initial-k-sweep/analysis.sh (hunk 1 above)
Comment (zrplumridge) [reply 1.initial-k-sweep/analysis.sh:19]: agreed

Comment (zrplumridge) [reply 1.initial-k-sweep/analysis.sh:19]: We should probably delete those from elsewhere in this module as well, such as in vae_pancancer.py and adage_pancancer.py

**Review by zrplumridge — APPROVED** [review]

**Pull request merged**